Makefile

```diff
@@ -109,7 +109,7 @@ vet: ## Run go vet on the whole project.
 	go vet ./...
 
 .PHONY: lint
-lint: bin/golangci-lint ## Run linting for the project.
+lint: bin/golangci-lint generate-mocks ## Run linting for the project.
```

Comment:
Will this result in the generate-mocks target being invoked twice when the test target is called and others that call lint?

Reply:
It should not. The targets for that are setup well enough that they won't regenerate if they're not older than their inputs.